teach_base/lesson_02/02_array_fibo.rb

```diff
@@ -0,0 +1,13 @@
+=begin
+Заполнить массив числами фибоначи до 100
+=end
+
+array_fibo = [0,1,1]
+next_num = array_fibo[-1] + array_fibo[-2]
+
+while next_num <= 100
+  array_fibo.push next_num
```

Comment:
Omitting braces is not a good practice
```suggestion
  array_fibo.push(next_num)
```
or
```suggestion
  array_fibo << next_num
```